integration/disk_flush_helpers.go

```diff
@@ -71,21 +101,43 @@ func waitUntilDataFlushed(
 
 func verifyForTime(
 	t *testing.T,
+	storageOpts storage.Options,
 	reader fs.FileSetReader,
 	shardSet sharding.ShardSet,
 	iteratorPool encoding.ReaderIteratorPool,
 	timestamp time.Time,
 	namespace ident.ID,
+	isSnapshot bool,
 	expected generate.SeriesBlock,
 ) {
+	fmt.Println("huh: ", timestamp.Unix())
```

Comment:
Want to remove this debugging statement?

Reply:
done